.github/workflows/test-notify-build-status.yml

```diff
@@ -2,16 +2,19 @@
 name: test-notify-build-status
 
 on:
-  workflow_dispatch:
   pull_request:
     paths:
       - '.github/actions/notify-build-status/**'
+  workflow_dispatch:
   push:
     branches:
       - main
     paths:
```

Comment:
Does it make sense to add the workflow itself?

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2256/commits/bd5129ee259a4d3d9bc95c14acc78775e2674b1a